app/code/Magento/Catalog/Test/Mftf/Test/AdminDeleteConfigurableChildProductsTest.xml

```diff
@@ -129,6 +134,11 @@
             <argument name="productName" value="$$createConfigProduct.name$$"/>
         </actionGroup>
         <!--Open Product Store Front Page and Verify Product is Out Of Stock -->
+        <amOnPage url="$$createConfigProduct.custom_attributes[url_key]$$.html" stepKey="openProductInStoreFront1"/>
+        <waitForPageLoad stepKey="waitForProductToLoad1"/>
+        <seeElement selector="{{StorefrontHeaderSection.NavigationCategoryByName($$createCategory.name$$)}}" stepKey="seeCategoryInFrontPage1"/>
+        <actionGroup ref="AssertStorefrontProductInfoMainProductNameActionGroup" stepKey="seeProductNameInStoreFront1">
+            <argument name="value" value="$$createConfigProduct.name$$"/>
```

Comment:
Hi @engcom-Charlie, looks like something went wrong on the last commit while merging the mainline. These changes shouldn't be here.

Reply:
HI @eduard13, thanks for updating. The merged issue has been fix now.